### PR TITLE
Remove 'field' word from nodes declaration

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Currently the nodes declaration is not consistent: some nodes use the 'field' world but most of them don't use it.

I removed it to make the declaration consistent.
It remains to decide what to do with 'hiddenField' words.

### CODE REVIEW

**Comment by omichel** [comment]
That was inspired from VRML97: https://www.evl.uic.edu/swami/VRML97/spec.DIS/part1/nodesRef.html
The idea was to have the `field` or `hiddenField` keyword in front of field names.

I am now wondering whether we could have a more compact representation: removing the `field` keyword and adding a dot prefix to hidden fields (like hidden files on Linux).

**Comment by stefaniapedrazzi** [comment]
Since the documentation is not automatically built from the WRL files, we can temporary keep the same format in WRL as now given that it works properly (low priority).

But in the documentation we can use a more compact representation and the word 'field' is not needed for the users.
Using a dot instead of 'hiddenField' in the documentation is fine for me.

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/inertialunit.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Derived from [Device](device.md).
 
 ```
 InertialUnit {
-  MFVec3f    lookupTable    []    # interpolation
+  MFVec3f    lookupTable    [ ]   # interpolation
   SFBool     xAxis          TRUE  # compute roll
   SFBool     zAxis          TRUE  # compute pitch
   SFBool     yAxis          TRUE  # compute yaw
```
Comment (omichel) [inline reference/inertialunit.md]: Note: the spacing is not consistent between the data type and the field name and between the field name and the default value and between the default value and the comment. Sometimes, we have a minimum amount of spaces (up to 1) and sometimes, we have an arbitrary number of spaces, like here 4 or 5.

Comment (stefaniapedrazzi) [inline reference/inertialunit.md]: Yes, I notice it too and there is no rule used to decide the spaces that is why I kept them as they are.

**Comment by omichel** [comment]
Also, I have some doubts about the dot notation for the hidden field. I am afraid that users may try to enter the dot in hidden fields in .wbt files and that won't work (unless we support it we could be a good idea...). At least it should be explained somewhere...

File: reference/inertialunit.md (hunk 1 above)
Comment (omichel) [inline reference/inertialunit.md]: We should fix that. Maybe in a different PR.

Comment (stefaniapedrazzi) [inline reference/inertialunit.md]: First we should define how to fix it. I will open a new issue.

**Comment by stefaniapedrazzi** [comment]
I remove the dot and instead wrote in the description that the field is hidden.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:

**Pull request merged**